cilium/cilium.go

```diff
@@ -0,0 +1,12 @@
+package cilium
```

Comment:
```suggestion
package vm
```
let's declare this under vm package only, `vm/cilium.go`

Reply:
Done.